### PR TITLE
Joke Telling Website using Vanilla JS #24

### PR DESCRIPTION
**Pull request by AftabMankapure**
# Pull Request for WebMasterLog 💡

## Issue Title:  Joke Telling Website using Vanilla JS

- **Name of the project** : Joke Telling website
- **Category:**  Vanilla-JS
- **Name:**  Aftab Amin Mankapure
- **GitHub ID:**  https://github.com/AftabMankapure
- **Idenitfy yourself:**  JWOC

Closes: #24 

### Describe the add-ons or changes you've made 📃
Added joke telling website using Vanilla-JS

Give a clear description of what have you added or modifications made

## Type of change ☑️

What sort of change have you made:

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested? ⚙️

Describe how it has been tested
Describe how have you verified the changes made

## Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added things that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

## Screenshots 📷

joke

### CODE REVIEW

**Comment by AftabMankapure** [comment]
pls change easy label to hard or medium

File: Vanilla-JS-Projects/Basic/Joke-Telling-Website/README.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+<h1 align='center'><b>💥 Joke Telling Website 💥</b></h1>
+
+<!-- -------------------------------------------------------------------------------------------------------------- -->
+
+<h3 align='center'>HTML, CSS, Vanilla-JS 🎮</h3>
```
Comment (Avdhesh-Varshney) [inline Vanilla-JS-Projects/Basic/Joke-Telling-Website/README.md:5]: remove this line.

File: Vanilla-JS-Projects/Basic/Joke-Telling-Website/README.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+<!-- enlist all the technologies used to create this project from them (Remove comment using 'ctrl+z' or 'command+z') -->
+
+<div align='center'>
+
+  <!-- ![HTML5](https://img.shields.io/badge/html5-%23E34F26.svg?style=for-the-badge&logo=html5&logoColor=white) -->
+  <!-- ![CSS3](https://img.shields.io/badge/css3-%231572B6.svg?style=for-the-badge&logo=css3&logoColor=white) -->
+  <!-- ![Bootstrap](https://img.shields.io/badge/bootstrap-%238511FA.svg?style=for-the-badge&logo=bootstrap&logoColor=white) -->
+  <!-- ![JavaScript](https://img.shields.io/badge/javascript-%23323330.svg?style=for-the-badge&logo=javascript&logoColor=%23F7DF1E) -->
```
Comment (Avdhesh-Varshney) [inline Vanilla-JS-Projects/Basic/Joke-Telling-Website/README.md:13]: and remove comments from here which tech stacks are used.

File: .vscode/settings.json
Comment (Avdhesh-Varshney) [inline .vscode/settings.json:1]: remove this file and directory

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]

File: Vanilla-JS-Projects/Basic/Joke-Telling-Website/README.md (hunk 1 above)
Comment (AftabMankapure) [reply Vanilla-JS-Projects/Basic/Joke-Telling-Website/README.md:5]: <h3 align='center'>HTML, CSS, Vanilla-JS 🎮</h3>

or

Comment (AftabMankapure) [reply Vanilla-JS-Projects/Basic/Joke-Telling-Website/README.md:5]: ---- which line

Comment (Avdhesh-Varshney) [reply Vanilla-JS-Projects/Basic/Joke-Telling-Website/README.md:5]: h3 heading line

File: .vscode/settings.json
Comment (AftabMankapure) [reply .vscode/settings.json:1]: which one

Comment (Avdhesh-Varshney) [reply .vscode/settings.json:1]: .vscode

File: Vanilla-JS-Projects/Basic/Joke-Telling-Website/README.md (hunk 1 above)
Comment (Avdhesh-Varshney) [reply Vanilla-JS-Projects/Basic/Joke-Telling-Website/README.md:5]: go through other projects directory and take reference ok

File: .vscode/settings.json
Comment (Avdhesh-Varshney) [inline .vscode/settings.json:1]: delete the .vscode directory ok

File: Vanilla-JS-Projects/Basic/Joke-Telling-Website/README.md (hunk 1 above)
Comment (Avdhesh-Varshney) [reply Vanilla-JS-Projects/Basic/Joke-Telling-Website/README.md:4]: check the other projects readme file how the tech stacks are written ok

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
@AftabMankapure after made these changes, mentioned to me.

**Comment by AftabMankapure** [comment]
@Avdhesh-Varshney  check this PR now

**Review by Avdhesh-Varshney — APPROVED** [review]
PR Approved 🎉

**Pull request merged**